CMakeLists.txt

```diff
@@ -107,7 +107,9 @@ include(Findnmodl)
 # ispc support
 if (ENABLE_ISPC_TARGET)
     enable_language(ISPC)
-    add_definitions("-DISPC_INTEROP=1")
+    if (CORENEURON_ISPC_INTEROP)
```

Comment:
I don't think this extra `if` statement is correct.

Reply:
Just interested in knowing, the variable(precisely flag/dir) should be inside an `if`, no?

Reply:
`CORENEURON_ISPC_INTEROP` CMake variable does not exist and is never set.
According to the code in [master branch](https://github.com/BlueBrain/CoreNeuron/blob/master/CMakeLists.txt#L108-L112), the `CORENEURON_ISPC_INTEROP` macro should be set if `ENABLE_ISPC_TARGET` CMake variable is True.

Reply:
:cool:

Reply:
:cool: ok, I just wanted that as an option as every CMake variable should be present as an option and under an `if`.